lms/djangoapps/discussion/config/settings.py

```diff
@@ -0,0 +1,22 @@
+"""
+Discussion settings.
+"""
+from django.conf import settings
+
+# .. toggle_name: FEATURES['ENABLE_FORUM_DAILY_DIGEST']
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: True
```

Comment:
See default questions below.  Also, note that we should understand how and when one should annotate the toggle definition vs setting toggle value (e.g. in envs).  Differences need to be noted for when the toggle is defined in the same repo, a different Open edX repo, or a third-party repo.